.circleci/config.yml

```diff
@@ -174,6 +198,29 @@ jobs:
       - deploy-to-aws:
           subdirectory: '/${MAJOR_VERSION}'
           cache-control: 'max-age=43200'
+  # deploys assets to a long-lived-cache folder in the S3 bucket named by release tag
```

Comment:
I was thinking it could be nice if we extracted the shared logic for deploy_version and deploy_search_bar_version, but think it's unneeded unless we plan to add additional "sub-libraries" (I'm making up words)

Reply:
We are planning on reworking the logic for deploying the major version since it doesn't take into account whether or not it's the latest version, so we could consolidate then